microsoft/testsuites/cvm/cvm_attestation.py

```diff
@@ -11,20 +11,66 @@
     TestSuite,
     TestSuiteMetadata,
 )
-from lisa.testsuite import TestResult
+from lisa.features.security_profile import CvmEnabled
+from lisa.operating_system import Ubuntu
+from lisa.sut_orchestrator.azure import features
+from lisa.testsuite import TestResult, simple_requirement
 from lisa.tools import Ls
 from lisa.util import SkippedException
-from microsoft.testsuites.cvm.cvm_attestation_tool import CVMAttestationTests
+from microsoft.testsuites.cvm.cvm_attestation_tool import (
+    AzureCVMAttestationTests,
+    NestedCVMAttestationTests,
+)
+
+
+@TestSuiteMetadata(
+    area="cvm",
+    category="functional",
+    description="""
+    This test suite is for generating CVM attestation report only for azure cvms.
```

Comment:
If the attestation report is not for Azure only, you don't need to explain for Azure here. If it is, add restriction in requirements like `simple_requirement(supported_platform_type=[AZURE])`,